python/dask_cudf/dask_cudf/batcher_sortnet.py

```diff
@@ -137,3 +146,164 @@ def sort_delayed_frame(parts, by):
     valid = compute(valid_ct)[0]
     validparts = parts[:valid]
     return validparts
+
+
+def set_partitions_hash(df, columns, npartitions):
+    c = hash_object_dispatch(df[columns], index=False)
+    return np.mod(c, npartitions)
+
+
+def _shuffle_group_2(df, columns, ignore_index):
+    if not len(df):
+        return {}, df
+    ind = hash_object_dispatch(df[columns], index=False)
+    n = ind.max() + 1
```

Comment:
Isn't `ind` here a Series / array of hashed values? `.max()` feels like it would be an arbitrary value, should this be `len(ind)`?

Reply:
Ah shoot - This is may not be tested.  This is a bad translation from the sort-based version of the same code in dask.dataframe.  We need to take a modulus of `ind` wrt `npartitions` (cant take the max) - I'll fix this